test/README.md

```diff
@@ -119,7 +122,7 @@ Debugging tests
   If `$VALGRIND` is also set it will pass `--vgdb=yes` to valgrind instead of
   starting gdbserver directly.
 
-  See [test/functional/testutil.lua](https://github.com/neovim/neovim/blob/9cadbf1d36b63f53f0de48c8c5ff6c752ff05d70/test/functional/testutil.lua#L52-L69) for details.
+  See [test/functional/nvimsession.lua](https://github.com/neovim/neovim/blob/9cadbf1d36b63f53f0de48c8c5ff6c752ff05d70/test/functional/nvimsession.lua#L52-L69) for details.
```

Comment:
This file doesn't not exist on this commit.

Reply:
And there is another instance in `runtime/doc/dev_tools.txt`

Reply:
I've removed them.